frontend_tests/node_tests/search_suggestion.js

```diff
@@ -464,17 +464,17 @@ run_test("has_suggestions", () => {
 
     query = "has:";
     suggestions = get_suggestions("", query);
-    expected = ["has:link", "has:image", "has:attachment"];
+    expected = ["has:", "has:link", "has:image", "has:attachment"];
```

Comment:
What's the background for why these tests change as a result of this PR?

Reply:
As now with this change we are treating invalid search operands as broken search so something like <code> has:</code> would also get a suggestion "Search for has:" .

Reply:
I think I should mention it in the commit.